src/app/create-event/page.tsx

```diff
@@ -101,27 +123,114 @@ export default function CreateEvent() {
         : JSON.parse('{"days": ["2024-01-01"]}'), // filler for specific mode now because no calendar yet
       '9e33186f-95db-4385-a974-ee38c8e07547',
     )
+
+    // Call handleSaveResponse to save the response
+    handleSaveResponse()
+  }
+
+  // Function to open modal for sign in
+  const openModal = () => {
+    if (dialogRef.current) {
+      dialogRef.current.showModal()
+    }
   }
 
   return (
-    <EventForm
-      title={title}
-      setTitle={setTitle}
-      description={description}
-      setDescription={setDescription}
-      location={location}
-      setLocation={setLocation}
-      earliestTime={earliestTime}
-      setEarliestTime={setEarliestTime}
-      latestTime={latestTime}
-      setLatestTime={setLatestTime}
-      mode={mode}
-      setMode={setMode}
-      daysOfWeek={daysOfWeek}
-      setDaysOfWeek={setDaysOfWeek}
-      timezone={timezone}
-      setTimezone={setTimezone}
-      handleSubmit={handleSubmit}
-    />
+    <div //main screen
+      className="flex min-h-screen w-full flex-col gap-8 p-8 md:flex-row"
+    >
+      <section //Left side container (Event form)
+        className="h-full w-full rounded-lg px-6 py-16 shadow-lg md:w-[28%]"
+      >
+        <EventForm
+          title={title}
+          setTitle={setTitle}
+          description={description}
+          setDescription={setDescription}
+          location={location}
+          setLocation={setLocation}
+          earliestTime={earliestTime}
+          setEarliestTime={setEarliestTime}
+          latestTime={latestTime}
+          setLatestTime={setLatestTime}
+          mode={mode}
+          setMode={setMode}
+          daysOfWeek={daysOfWeek}
+          setDaysOfWeek={setDaysOfWeek}
+          timezone={timezone}
+          setTimezone={setTimezone}
+        />
+
+        <div //button container for positioning button
+          className="mx-4 flex justify-center pt-8"
+        >
+          <button
+            className="btn btn-primary ml-4 rounded-full px-4 py-2 text-white"
+            onClick={openModal}
+          >
+            Add Availability
+          </button>
+
+          <dialog ref={dialogRef} id="my_modal_1" className="modal">
```

Comment:
maybe a more descriptive id here? Is this the Username modal? maybe `username_modal` ?

Reply:
Working on this, should be fixed in the next commit